skyportal/handlers/api/photometry.py

```diff
@@ -181,14 +182,22 @@ def post(self):
             group_ids = data.pop("group_ids")
         except KeyError:
             return self.error("Missing required field: group_ids")
-        groups = Group.query.filter(Group.id.in_(group_ids)).all()
-        if not groups:
-            return self.error(
-                "Invalid group_ids field. " "Specify at least one valid group ID."
-            )
-        if not all([group in self.current_user.accessible_groups for group in groups]):
+        if isinstance(group_ids, (list, tuple)):
+            groups = Group.query.filter(Group.id.in_(group_ids)).all()
+            if not groups:
+                return self.error(
+                    "Invalid group_ids field. Specify at least one valid group ID."
+                )
+            if not all(
+                [group in self.current_user.accessible_groups for group in groups]
+            ):
+                return self.error(
+                    "Cannot upload photometry to groups that you are not a member of."
+                )
```

Comment:
```suggestion
```

Reply:
I don't think we need to enforce this. Sometimes bots will upload photometry to groups they are not members of. That should be allowed

Reply:
Note that this is not new to this PR, it was just being moved. But I can remove it here if you'd like

Reply:
Done

Reply:
thanks